src/main/kotlin/app/revanced/patches/remini/misc/premium/patch/PremiumUnlockPatch.kt

```diff
@@ -0,0 +1,38 @@
+package app.revanced.patches.remini.misc.premium.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.MethodFingerprintExtensions.name
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultError
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.remini.misc.premium.annotations.PremiumUnlockCompatibility
+import app.revanced.patches.remini.misc.premium.fingerprints.PremiumUnlockFingerprint
+
+@Patch
+@Name("premium-unlock")
```

Comment:
```suggestion
@Name("unlock-premium")
```

Make sure to also rename the class accordingly.